scripts/parameters.ts

```diff
@@ -1,14 +1,61 @@
 export const RELEASE_TIME: number = Math.floor(new Date("2021.04.21 15:00:00 GMT").getTime() / 1000)
 
 // it's a test only allocations with a random addresses!
+
+// @note: All values is set without decimals
+
+// TODO: Just curious - why index: string, not index: number?
```

Comment:
Because js automatically convert keys to strings under the hood.

cc @idealatom

Reply:
In js keys can only be strings, and numeric keys are coerced and stored as strings.